agent/consul/acl_replication.go

```diff
@@ -453,23 +478,28 @@ func (s *Server) replicateACLTokens(lastRemoteIndex uint64, ctx context.Context)
 
 	// Calculate the changes required to bring the state into sync and then
 	// apply them.
-	deletions, updates := diffACLTokens(local, remote.Tokens, lastRemoteIndex)
-	s.logger.Printf("[DEBUG] acl: token replication - deletions: %d, updates: %d", len(deletions), len(updates))
+	res := diffACLTokens(local, remote.Tokens, lastRemoteIndex)
+	if res.LocalSkipped > 0 || res.RemoteSkipped > 0 {
+		s.logger.Printf("[DEBUG] acl: token replication - deletions: %d, updates: %d, skipped: %d, skippedRemote: %d",
```

Comment:
Since the skipped entries will mostly not happen, I didn't want to clutter this debug with two things that may always be zero for most people post-upgrade.